From 3b43532c2ea450b28cfdbd238af43571cc917cd4 Mon Sep 17 00:00:00 2001
From: Marek 'marx' Grac <mgrac@redhat.com>
Date: Wed, 18 Feb 2015 14:25:43 +0100
Subject: [PATCH 15/16] fence_vmware_soap: Remove validitation of certificates

Upstream/RHEL7 uses python-requests module which is not available.
---
 fence/agents/vmware_soap/fence_vmware_soap.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/fence/agents/vmware_soap/fence_vmware_soap.py b/fence/agents/vmware_soap/fence_vmware_soap.py
index 5f760cc..ebd3491 100644
--- a/fence/agents/vmware_soap/fence_vmware_soap.py
+++ b/fence/agents/vmware_soap/fence_vmware_soap.py
@@ -2,7 +2,7 @@
 
 import sys
 import shutil, tempfile, suds
-import logging, requests
+import logging
 import atexit
 sys.path.append("@FENCEAGENTSLIBDIR@")
 
@@ -58,7 +58,7 @@ def soap_login(options):
 
 	try:
 		headers = {"Content-Type" : "text/xml;charset=UTF-8", "SOAPAction" : ""}
-		conn = Client(url + "/vimService.wsdl", location=url, transport=RequestsTransport(verify=verify), headers=headers)
+		conn = Client(url + "/vimService.wsdl", location=url, headers=headers)
 
 		mo_ServiceInstance = Property('ServiceInstance')
 		mo_ServiceInstance._type = 'ServiceInstance'
-- 
1.9.3

